swift/iOS/Chat/ChatLayoutController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  ChatLayoutController.swift
+//  IOSChat
+//
+//  Created by Reece Humphreys on 6/18/19.
+//
+
```

Comment:
You should add ZeroC copyright headers to all new files

Reply:
Sorry about that, forgot to remove the default copyright header